base58.py

```diff
@@ -1,20 +1,21 @@
-#!/usr/bin/env python
+'''Base58 encoding
 
-# from:
-# https://raw.githubusercontent.com/gavinandresen/bitcointools/f377bbe0882669423895c6e91b163d93d4fdf560/base58.py
-
-# Copyright (c) 2010 Gavin Andresen
+Implementations of Base58 and Base58Check endcodings that are compatible
+with the bitcoin network.
+'''
 
+# Copyright (c) 2015 David Keijser
```

Comment:
Is this now a combination of the old script and https://github.com/trezor/trezor-core/blob/master/src/trezor/crypto/base58.py ?